tests/tensorflow/r2.12/common.libsonnet

```diff
@@ -236,7 +236,7 @@ local mixins = import 'templates/mixins.libsonnet';
     schedule: functional_schedule,
   },
   Convergence:: mixins.Convergence {
-    schedule: null,
+    schedule: '0 11 * * *',
```

Comment:
IIUC this is saying to run convergence daily at 11? Should we do every other day instead?